src/main/java/build/buildfarm/server/ByteStreamService.java

```diff
@@ -262,9 +262,11 @@ private void writeOperationStream(
         String operationStream = parseOperationStream(resourceName);
 
         OutputStream outputStream = instance.getStreamOutput(operationStream);
-        outputStream.write(request.getData().toByteArray());
-        if (request.getFinishWrite()) {
-          outputStream.close();
+        if (outputStream != null) {
```

Comment:
This got merged in as a part of continuing work where I didn't have an implementation of getStreamOutput (it returned null in that case).  I will drop the check, as it shouldn't happen with a properly implemented instance.  I've toyed with the idea of a stream enabled feature/config, if I put that in, I'll put a check at a level higher than this.